fk/luse/src/main/java/se/inera/intyg/common/luse/v1/validator/InternalDraftValidatorImpl.java

```diff
@@ -324,44 +329,44 @@ private void validateKontaktMedFk(LuseUtlatandeV1 utlatande, List<ValidationMess
         // R11
         if ((utlatande.getKontaktMedFk() == null || !utlatande.getKontaktMedFk())
             && !Strings.nullToEmpty(utlatande.getAnledningTillKontakt()).trim().isEmpty()) {
-            ValidatorUtil.addValidationError(validationMessages, CATEGORY_KONTAKT, KONTAKT_ONSKAS_SVAR_JSON_ID_26,
+            ValidatorUtil.addValidationErrorWithQuestionId(validationMessages, CATEGORY_KONTAKT, KONTAKT_ONSKAS_SVAR_JSON_ID,
                 ValidationMessageType.INCORRECT_COMBINATION,
-                "luse.validation.kontakt.incorrect_combination");
+                "luse.validation.kontakt.incorrect_combination", KONTAKT_ONSKAS_SVAR_ID);
         }
     }
 
     private void validateBlanksForOptionalFields(LuseUtlatandeV1 utlatande, List<ValidationMessage> validationMessages) {
```

Comment:
Should question ids be added to error messages in this method as well (I did)

Reply:
I think you are absolutely right that we should also decorate those validation messages with questionId. The reason I didn't initially was because when I tested how the angular client handled it, it didn't show any errors. But it seems that it still serves some function in angular and they actually remove all blank spaces when signing.

Updated.